### PR TITLE
Update --no-server-version-check  option description in 8.1

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
modified:   docs/server-backup-version-comparison.md

### CODE REVIEW

File: docs/server-backup-version-comparison.md [hunk 1]
```diff
@@ -1,12 +1,10 @@
-<!--- will this be in 8.1?
- --->
-
 # Server version and backup version comparison
 
 A MySQL change to a feature, such as the structure of a redo log
 record, can cause older versions of Percona XtraBackup to fail. To
 ensure that you can back up and restore your data, use a Percona
-XtraBackup version that is equal to or above your source server version.
+XtraBackup version that is equal to or within your source server major version. For example, if you use Percona
```
Comment (patrickbirch) [inline docs/server-backup-version-comparison.md]: Maybe add a reminder that 8.1 only works with 8.1? 

equal to or within your source

within?

File: docs/server-backup-version-comparison.md [hunk 2]
```diff
@@ -36,6 +34,8 @@ example, overrides the parameter and the backup proceeds.
 $ xtrabackup --backup --no-server-version-check --target-dir=$mysql/backup1
 ```
 
+Overriding the `--no-server-version-check` parameter allows taking backups using a Percona XtraBackup version that is equal to or greater than your source server major version but up to the next LTS release. For example, if you use Percona XtraBackup 8.1.x, you can back up the source system from 8.1.x to 8.4.xx.
```
Comment (patrickbirch) [inline docs/server-backup-version-comparison.md]: remove the "but" 

major version of your source server up to the last release in the LTS series. 

Maybe?

you can back up the source server from 8.1.x to 8.4.x. 

We call it "source server" in the sentence before this.

File: docs/server-backup-version-comparison.md (hunk 1 above)
Comment (alina-derkach-oaza) [reply docs/server-backup-version-comparison.md]: Updated

File: docs/server-backup-version-comparison.md (hunk 2 above)
Comment (alina-derkach-oaza) [reply docs/server-backup-version-comparison.md]: Updated

File: docs/server-backup-version-comparison.md [hunk 3]
```diff
@@ -1,33 +1,30 @@
...
 
 !!! admonition "See also"
    
     [How XtraBackup works](how-xtrabackup-works.md)
 
 The `--no-server-version-check` option performs the following test.
-Before the backup starts, XtraBackup compares the source system version to
-the Percona XtraBackup version. If the source system version is greater
+Before the backup starts, XtraBackup compares the source server version to
+the Percona XtraBackup version. If the source server version is greater
```
Comment (patrickbirch) [inline docs/server-backup-version-comparison.md:14]: This is confusing.
In the first paragraph, we say they can safely back up the source server from 8.1.x t0 8.1.xx
Here, we say that if the source server is greater than the PXB version xtrabackup stops the backup process
At the end we say that if they override the check they can back up to 8.4.xx

Comment (alina-derkach-oaza) [reply docs/server-backup-version-comparison.md:14]: updated

File: docs/server-backup-version-comparison.md [hunk 4]
```diff
@@ -1,33 +1,30 @@
...
 
 !!! admonition "See also"
    
     [How XtraBackup works](how-xtrabackup-works.md)
 
 The `--no-server-version-check` option performs the following test.
-Before the backup starts, XtraBackup compares the source system version to
-the Percona XtraBackup version. If the source system version is greater
-than the XtraBackup version, XtraBackup stops the backup and returns an
+Before the backup starts, XtraBackup compares the source server version to
```
Comment (altmannmarcelo) [inline docs/server-backup-version-comparison.md]:
```suggestion
Before the backup/prepare starts, XtraBackup compares the source server version to
```

**Review by altmannmarcelo — CHANGES_REQUESTED** [review]
one minor comment. Same check is now used on prepare too

**Review by altmannmarcelo — APPROVED** [review]
LGTM

File: docs/server-backup-version-comparison.md [hunk 5]
```diff
@@ -1,33 +1,30 @@
-<!--- will this be in 8.1?
- --->
-
 # Server version and backup version comparison
 
 A MySQL change to a feature, such as the structure of a redo log
 record, can cause older versions of Percona XtraBackup to fail. To
 ensure that you can back up and restore your data, use a Percona
-XtraBackup version that is equal to or above your source server version.
+XtraBackup version that is equal to your source server major version. This means if you use Percona XtraBackup 8.1.x, you can safely back up the source server from 8.1.x to 8.1.xx.
```
Comment (patrickbirch) [inline docs/server-backup-version-comparison.md:6]: 8.1.x to 8.1.xx?

**Review by patrickbirch — APPROVED** [review]
OK

**Pull request merged**